swagger/v1/component_schemas/ECFParticipantAttributes.yml

```diff
@@ -70,12 +70,13 @@ properties:
     type: boolean
     nullable: true
     example: true
-  state:
-    description: "The state of the ECF participant"
+  training_status:
+    description: "The training status of the ECF participant"
```

Comment:
Can we have it in its own little PR please? I would like this to go in as quick as possible, and this PR is large and complex :/

Reply:
Done. This one adds in the "defer" status as well though, which isn't in the previous PR, so it'll still be a (smaller) change in this PR.